src/test/java/reposense/git/GitShortlogTest.java

```diff
@@ -16,39 +17,77 @@
 public class GitShortlogTest extends GitTestTemplate {
 
     @Test
-    public void extractAuthorsFromLog_validRepoNoDateRange_success() {
+    public void getShortlogSummary_noDateRange_success()
+            throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+        Method m = GitShortlog.class.getDeclaredMethod("getShortlogSummary", String.class, Date.class, Date.class);
+        m.setAccessible(true);
+        String result = (String) m.invoke(null, config.getRepoRoot(), null, null);
```

Comment:
Rather than using Reflection to test the `private` helper method, we can test the `public` method instead `GitShortlog.getAuthors(...)` with a `RepoConfiguration` with the specified values instead?

Reply:
Hmm true, wrote the test without realizing that. Will remove them.